import-export-cli/integration/apim/apiDTO.go

```diff
@@ -242,10 +242,12 @@ type APIOperations struct {
 
 // AdvertiseInfo : Advertise only information
 type AdvertiseInfo struct {
-	Advertised           bool   `json:"advertised" yaml:"advertised"`
-	OriginalDevPortalUrl string `json:"originalDevPortalUrl,omitempty" yaml:"originalDevPortalUrl,omitempty"`
-	ApiOwner             string `json:"apiOwner,omitempty" yaml:"apiOwner,omitempty"`
-	Vendor               string `json:"vendor,omitempty" yaml:"vendor,omitempty"`
+	Advertised                    bool   `json:"advertised,omitempty" yaml:"advertised,omitempty"`
```

Comment:
shall remove `,omitempy` in both places? otherwise it causes to lose the default boolean value.

Reply:
Done with https://github.com/wso2/product-apim-tooling/pull/863/commits/2df19483835178b65d6c9457457a34f04bb774b3